roles/zookeeper/tasks/main.yml

```diff
@@ -81,4 +80,24 @@
 
 - include: collectd.yml
 
+- name: stop zookeeper service
+  sudo: yes
+  service:
+    name: "{{ zookeeper_service }}"
+    state: stopped
+  when: repair_mode is defined
+  tags:
+    - zookeeper
+
+- name: register zookeeper service
+  sudo: yes
+  service:
+    enabled: yes
+    name: "{{ zookeeper_service }}"
+    state: started
+  notify:
+    - wait for zookeeper to listen
+  tags:
+    - zookeeper
+
```

Comment:
We don't "start" it every time.  We just check whether it's started and if it's not, then we start it (once).